docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,138 @@
+The callbacks `clipboardPasteStart` and `clipboardPasteEnd` are fired during the clipboard paste operation, which can be useful for extra customizations around the event.
+
+For more details on how to use clipboard copy and paste, check out the [Data Grid—Clipboard documentation](https://mui.com/x/react-data-grid/clipboard/).
+
+### Filter on column headers[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+Filtering on column headers gives users the ability to quickly filter data without any additional menu.
+Use the the `unstable_headerFilters` prop to activate this feature.
+
+The new filter fields are displayed below the headers, and are synchronized with the filter panel.
+If you prefer more simplicity, you can disable the default filter panel using the `disableColumnFilter` prop, and set filters to use only the default operator.
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/mui-x-mid-v6-features/header_filter.mp4" type="video/mp4">
+</video>
+
+For more details on how to use and customize the filters, check out its [documentation page](https://mui.com/x/react-data-grid/filtering/header-filters/).
+
+### Performance enhancements: smoother scrolling and expanding rows
+
+The Data Grid received significant performance improvements in MUI X v6, particularly concerning its scrolling efficiency and the expansion of child rows.
+
+These performance upgrades are part of our ongoing commitment to provide end users with a superior experience.
+We always strive to ensure that MUI X remains performant and efficient, particularly when dealing with large and complex data.
+
+## Charts - alpha version 🎁
+
+We are delighted to announce the upcoming expansion of MUI X: a brand-new suite of components for building and customizing charts.
+
+With MUI X Charts, you can choose from a wide range of chart types, including line charts, bar charts, pie charts, area charts, scatter plots, and more. 
+Each chart type is thoughtfully crafted with attention to detail, ensuring that the visual representations are not only aesthetically pleasing, but also highly effective in conveying complex data.
+
+And it ships with gorgeous palettes that were specially crafted by our designers!
+
+Check out the video below, highlighting some of our Charts:
+
+<video preload="metadata" autoplay muted loop playsinline controls>
+  <source src="/static/blog/mui-x-mid-v6-features/charts_final_v1.mov" type="video/mp4">
+</video>
+
+As we progress toward the stable version, we're committed to enhancing the overall experience for developers and users, so your feedback is fundamental!
+
+We will also steadily expand our portfolio with new chart types such as [Heat Map](https://mui.com/x/react-charts/heat-map/),[Funnel](https://mui.com/x/react-charts/funnel/), [Gantt](https://mui.com/x/react-charts/gantt/), and more.
```

Comment:
It looks like we went with lower case on the page and GitHub issue, also missing space.

```suggestion
We will also steadily expand our portfolio with new chart types such as [Heat map](https://mui.com/x/react-charts/heat-map/), [Funnel](https://mui.com/x/react-charts/funnel/), [Gantt](https://mui.com/x/react-charts/gantt/), and more.
```

Reply:
yeah, I'm getting confused because I thought all component names should be capitalized. Like Date Pickers, Data Grid, etc. Wouldnt this be in the same bucket? Or no since it's a type under Charts? We should definitely standardize.

Reply:
will commit your suggestion